src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -333,72 +447,130 @@ public abstract class BoostedTreeArgs : TreeArgs
         // REVIEW: TLC FR likes to call it bestStepRegressionTrees which might be more appropriate.
         //Use the second derivative for split gains (not just outputs). Use MaxTreeOutput to "clip" cases where the second derivative is too close to zero.
         //Turning BSR on makes larger steps in initial stages and converges to better results with fewer trees (though in the end, it asymptotes to the same results).
+        /// <summary>
+        /// Use best regression step trees?
+        /// </summary>
```

Comment:
Same here if you could make this in a declarative sentence that would be better I think.

Reply:
fixed

---
In reply to: [245964319](https://github.com/dotnet/machinelearning/pull/2047#discussion_r245964319) [](ancestors = 245964319)